simpleclient_guava/src/main/java/io/prometheus/client/cache/guava/CacheMetricsCollector.java

```diff
@@ -0,0 +1,79 @@
+package io.prometheus.client.cache.guava;
```

Comment:
I considered that. I can't find any additional Guava items that might be added (but I'm happy to be proven wrong). 

Also I have `cache.caffeine` ready (I'll submit the PR after this one is done) and `cache.ehcache` soon after that. So it ay be a little weird to have a `guava.cache` that doesn't parallel those other 2.